back-end/Service.py

```diff
@@ -0,0 +1,280 @@
+    return jsonify(image_handler.add_image(image))
+    
+
+@app.route('/getAllImageTypes') 
+def get_all_image_types():
+    db = DataAccess()
+    return jsonify(db.get_all_image_types())
+
+
+@app.route('/getAllEventTypes') 
+def get_all_event_types():
+    db = DataAccess()
+    return jsonify(db.get_all_event_types())
+ 
+
+@app.route('/getAllImages') 
+def get_all_images():
+    db = DataAccessImage()
+    images_from_DB = db.get_all_images()
+    print(images_from_DB)
+    for image in images_from_DB:
+         image['path']=ConfigProvider.PHOTOS_BASE_URL+image['name']
+    resp = make_response(jsonify(images_from_DB, 200))
+    return resp
+
+@app.route('/updateImage', methods=['POST'])
+def update_image():
+    data = json.loads(request.stream.read())
+    db = DataAccessImage()
+    image_handler = ImageHandler(db)
+    image =  Image(data["name"], data["imageType"], data["eventType"], data["location"])
+    return jsonify(image_handler.edit_image(image))
+ 
+
+@app.route('/delete', methods=['POST'])
+def delete_image( ):
+    data = json.loads(request.stream.read())
+    db = DataAccessImage()
+    image_handler = ImageHandler(db)
+    image = Image(data["name"],data["imageType"],data["eventType"],data["location"])
+    return jsonify(image_handler.delete_image(image)) #RECHECK 
+
+
+@app.route('/addTimeUnit', methods=['POST'])
+def add_time_to_calendar():
+    data = json.loads(request.stream.read())
+    db = DataAccessCalendar()
+    tu_handler = TimeUnitHandler(db)
+    time_unit = TimeUnit(data["date"],data["dayOfWeek"],data["time"],data["isWeekly"] )
+    return jsonify(tu_handler.add_time_to_calendar(time_unit))
+
+@app.route('/deleteTimeUnit', methods=['POST'])
+def delete_time_from_calendar():
+    data = json.loads(request.stream.read())
+    db = DataAccessCalendar()
+    tu_handler = TimeUnitHandler(db)
+    timeunit = TimeUnit(data["date"],data["dayOfWeek"],data["time"],data["isWeekly"] )
+    return jsonify(tu_handler.delete_time_from_calendar(timeunit))
+
+@app.route('/deleteOrder', methods=['POST'])
+def delete_order():
+    data = json.loads(request.stream.read())
+    db = DataAccessOrders()
+    or_handler = OrderHandler(db)
+    return jsonify(or_handler.delete_order(data))
+             
+@app.route('/deleteLocationType', methods=['POST'])
+def delete_location_type():
+    data = json.loads(request.stream.read())
+    db = DataAccessLocation()
+    loc_handler = LocationHandler(db)
+    return jsonify(loc_handler.delete_location_type(data))
+
+
+@app.route('/updateOrder', methods=['POST'])
+def update_order():
+    data = json.loads(request.stream.read())
+    db = DataAccessOrders()
+    or_handler = OrderHandler(db)
+    order = Order( data["date"], data["time"],data["name"],data["telefon"],data["email"],data["location"], data["eventType"],data["note"], data["id"])
+    return jsonify(or_handler.update_order(order))
+
+@app.route('/getTimeSlots') 
+def get_all_time_slots():

```

Comment:
consider something unified for the "get all".....

Reply:
Don't understand